### DISCUSSION TITLE
Better support for JLink

### DISCUSSION THREAD

**Discussion by hendrikebbers** [support-and-care/maven-support-and-care#36] [Ideas for Funding]
This is based on the discussion in https://github.com/OpenElements/maven-support-care/discussions/11

Goal should be an "official" jlink support/plugin for maven

**Comment by rmannibucau** [comment]
https://maven.apache.org/plugins/maven-jlink-plugin/ ? 🤔

**Reply by hendrikebbers** [reply]
Good point. I assumed that this is a "not good supported 3rd party plugin" but never checked. It was part of a discussion I had with @desruisseaux so maybe I miss something

**Reply by rmannibucau** [reply]
this is an official apache maven plugin

**Reply by desruisseaux** [reply]
I was not aware of this plugin. It will need to be examined after the other plugins, in order to determine if there is work to do for better integration.
